### PR TITLE
doc(agent_with_memory_in_db):Get rid of ZeroShotAgent and use create_react_agent instead

### PR DESCRIPTION
**Pull request by liugddx**
- **Issue:** #20122
- @baskaryan,@eyurtsev

### CODE REVIEW

File: docs/docs/modules/memory/agent_with_memory_in_db.ipynb [hunk 1]
```diff
@@ -320,7 +297,7 @@
     }
    ],
    "source": [
-    "agent_without_memory.run(\"what is their national anthem called?\")"
+    "agent_executor__without_memory.run({\"input\": \"what is their national anthem called?\"})"
```
Comment (leo-gan) [inline docs/docs/modules/memory/agent_with_memory_in_db.ipynb:300]: Isn't it `.invoke(`?

Comment (liugddx) [reply docs/docs/modules/memory/agent_with_memory_in_db.ipynb]: > Isn't it `.invoke(`?

Fixed. PTAL

**Comment by liugddx** [comment]
@ccurme ptal

**Review by leo-gan — APPROVED** [review]
lgtm

**Review by baskaryan — COMMENTED** [review]
nit: could we initialize the `AgentExecutor(..., verbose=True)` so the outputs show the agent steps

**Review by baskaryan — APPROVED** [review]

**Pull request merged**